guardian/pluto-storagetier
# Fix/common upload

## What does this change?

Deprecates direct stream upload via Alpakka, as it is not working properly with ObjectLock protected buckets. Instead, use the S3 transfer library

## How to test

Deploy and then ingest media to Vidispine.  The alpakka upload was used for metadata/thumbs and was erroring and retrying in the logs. You should see them succeed

## How can we measure success?

Less errors and DLQs
